packages/styletron-react/src/types.js

```diff
@@ -35,14 +35,28 @@ export type Styletron = {
   reducers: Array<ReducerContainer>,
   base: any,
   driver: any,
+  name?: string,
   wrapper: any,
   getInitialStyle: any,
+  ext?: {
+    name?: string,
```

Comment:
Is this `name` property storing the same thing as the `name` property one level up? Or is it some different value. I suppose if it's the same, no need to have it twice? I suppose if `name` is only used by the extension, then it should go under `ext`?

Same thing with `getInitialStyle` and `base`.

Reply:
They aren't storing the same thing as the properties one level up 👍 
They store the values of the component that is being extended. 
It's used at both levels.